src/app/loadout-builder/LoadoutBuilder.tsx

```diff
@@ -249,12 +252,25 @@ function LoadoutBuilder({
       </PageWithMenu.Menu>
 
       <PageWithMenu.Contents>
-        {filteredSets && processing && (
-          <div className={styles.processing}>
-            <div>{t('LoadoutBuilder.ProcessingSets', { character: selectedStore.name })}</div>
-            <AppIcon icon={refreshIcon} spinning={true} />
-          </div>
-        )}
+        <TransitionGroup component={null}>
```

Comment:
Think it would be worth making this its own component? As a side note I want to organise some of the files in loadoutbuilder a little at some point.